### PR TITLE
Front end of comments 2.0

### PR DESCRIPTION
**Pull request by MandyMeindersma**
lol, I can't reopen my old one

### CODE REVIEW

**Comment by MandyMeindersma** [comment]
I am having issues with it actually posting to the api? so it's still not done but almost

File: react/src/js/react/posts/PostView.jsx [hunk 1]
```diff
@@ -1,25 +1,87 @@
...
+                </div>
+            </Thumbnail>
+
+        ));
+        return comments;
+    }
+
+    renderMakeComments() {
+        return (
+            <Form onSubmit={this.handleCommentSubmit}>
```
Comment (amalik2) [inline react/src/js/react/posts/PostView.jsx]: This method should probably return the loading component while a comment is being made

File: react/src/js/react/posts/PostView.jsx [hunk 2]
```diff
@@ -1,25 +1,87 @@
...
  */
 export default class PostView extends Reflux.Component {
     constructor(props) {
         super(props);
-        this.store = PostsStore;
+        this.stores = [PostsStore, AuthStore];
     }
 
     componentDidMount() {
         PostsActions.getPost(this.props.match.params.id);
```
Comment (amalik2) [inline react/src/js/react/posts/PostView.jsx]: You need to load more comments from the database here. The post by default only contains the 5 most recent ones. We need pagination style comment loading though. On initial mount, load the first 25 or whatever, then you need to add in a way to load more once the user scrolls to the bottom of the comments list (IF there is more)

File: react/src/js/react/posts/PostView.jsx [hunk 3]
```diff
@@ -1,25 +1,87 @@
...
+
+        ));
+        return comments;
+    }
+
+    renderMakeComments() {
+        return (
+            <Form onSubmit={this.handleCommentSubmit}>
+                <FormGroup controlId="comment">
+                    <FormControl bsSize="sm" name="comment" type="text" placeholder="Comment here..." />
```
Comment (amalik2) [inline react/src/js/react/posts/PostView.jsx]: Shouldn't this be a textarea? Comments can be pretty large. With that in mind, you should also put a max length of something reasonable like 500 characters or something

**Comment by MandyMeindersma** [comment]
image

**Comment by MandyMeindersma** [comment]
Adil also said:

Looks alright, but the Make comment function should return a <Thumbnail> wrapping everything
The type selection should be radio buttons, not a select component

**Pull request merged**